editor/test/editor/field_expression_test.clj

```diff
@@ -114,7 +114,6 @@
     (is (= 1.1 (to-double "1.1")))
     (is (= -2.0 (to-double "-2")))
     (is (= -2.5 (to-double "-2.5")))
-    (is (= -1.5 (to-double "-1,5")))
```

Comment:
Hmm, yes. I think it is probably fine to not allow commas in separators, so it should be fine to remove this test. 🤔

We added support for this quite some time ago, but I can't remember if we did it on a whim or if our users requested it. I suppose we can silently remove support for commas and await the response from the community. 😅